MyStoreAtomation/CatalogTests.cs

```diff
@@ -0,0 +1,156 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Infrastructure;
+using FluentAssertions;
+
+//ReferenceTypeAssertions ==> for later assertions
+
+namespace MyStoreAtomation
+{
+    [TestClass]
+    public class CatalogTests : UnitTestBase
+    {
+        [TestMethod]
+        public void AddToCartButtonPopsTest()
+        {
+            var threwExeption = false;
+            try
```

Comment:
the try catch is unnecessary. the test will fail if the exception was thrown and not caught.